requirements.txt

```diff
@@ -5,7 +5,7 @@ setuptools>=11.3
 
 # These are direct dependencies of ooniprobe.
 PyYAML>=3.10
-Twisted>=13.2.0
+Twisted<=18.7.0
```

Comment:
You need to still put the lower-bound dependency constraint, this can be done with:
```
Twisted>=13.2.0,<=18.7.0
```